test/common/schemaform/definitions/dateRange.unit.spec.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-addons-test-utils';
+import Form from 'react-jsonschema-form';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import { schema, uiSchema } from '../../../../src/js/common/schemaform/definitions/dateRange';
+
+describe('Schemaform definition dateRange', () => {
+  it('should render dateRange', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          uiSchema={uiSchema()}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    expect(formDOM.querySelectorAll('label').length).to.equal(8);
+    expect(formDOM.querySelectorAll('input').length).to.equal(2);
```

Comment:
Yeah, that would be probably be better. I was having trouble figuring out where to draw the line for these tests. I don't want to test every detail of all the elements rendered, but I want some assurance the right elements are present.